omod/src/main/java/org/openmrs/module/locationbasedaccess/fragment/controller/field/LocationsFragmentController.java

```diff
@@ -0,0 +1,21 @@
+package org.openmrs.module.locationbasedaccess.fragment.controller.field;
+
+import org.openmrs.Location;
+import org.openmrs.ui.framework.fragment.FragmentModel;
+
+import org.openmrs.api.context.Context;
+import javax.servlet.http.HttpSession;
+import java.util.List;
+
+/**
+ * Fields and actions to provide registrationInfo view.
```

Comment:
Does the above description add any value to the class?

Reply:
Removed